### PR TITLE
Port svml patch to 9.0.0

### PR DESCRIPTION
**Pull request by Hardcode84**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Based on #72

### CODE REVIEW

**Comment by Hardcode84** [comment]
@anton-malakhov

File: recipe/D47188-svml-VF.patch [hunk 1]
```diff
@@ -643,15 +551,15 @@ index c45dee5..4521fc9 100644
    /// The original loop.
    Loop *OrigLoop;
  
-@@ -4106,6 +4127,7 @@ void InnerLoopVectorizer::widenInstruction(Instruction &I) {
+@@ -4153,6 +4174,7 @@ void InnerLoopVectorizer::widenInstruction(Instruction &I) {
        }
  
        Function *VectorF;
 +      bool FromSVML = false;
```
[the remark this answers is not in the document]
Comment (Hardcode84) [reply recipe/D47188-svml-VF.patch]: That's from original patch, I didn't change this line. Probably github parser just got broken trying to show path for a patch.

**Comment by Hardcode84** [comment]
Tried to enable all llvm tests:

osx:
```
[  FAILED  ] FileSystemTest.permissions
```

linux:
```
Cannot find `gcc' in PATH: No such file or directory
```

**Comment by Hardcode84** [comment]
`llvm-lit -vv Transforms ExecutionEngine Analysis CodeGen/X86` test seems to work

**Comment by Hardcode84** [comment]
Tests on windows passing too.

Removed partial-testing.patch.

I will be on vacation next week so someone else will need to update URLs once 9.0.0 final is out.

File: recipe/D47188-svml-VF.patch [hunk 2]
```diff
@@ -1,9 +1,9 @@
-From 47ea18840aa0364a969073ef71d08c18fbc044b3 Mon Sep 17 00:00:00 2001
-From: Anton Malakhov <Anton.Malakhov@intel.com>
```
Comment (isuruf) [inline recipe/D47188-svml-VF.patch]: You should give credit in the patch description.

**Comment by isuruf** [comment]
@msarahan, looks like vs2017 activation scripts are broken.

**Comment by isuruf** [comment]
Thanks @Hardcode84, @anton-malakhov

**Comment by msarahan** [comment]
@isuruf need more info.  Please don't send us on a goose chase.  What is the error?  When did it start?  Does it happen everywhere or only a few recipes?

**Comment by isuruf** [comment]
Log has this
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise>CALL "VC\Auxiliary\Build\vcvars64.bat" -vcvars_ver=14.16 wdf 
[ERROR:vcvarsall.bat] Invalid argument found : wdf
[ERROR:vcvarsall.bat] Error in script usage. The correct usage is:
Syntax:
    vcvarsall.bat [arch] [platform_type] [winsdk_version] [-vcvars_ver=vc_version] [-vcvars_spectre_libs=spectre_mode]
where : 
    [arch]: x86 | amd64 | x86_amd64 | x86_arm | x86_arm64 | amd64_x86 | amd64_arm | amd64_arm64
    [platform_type]: {empty} | store | uwp 
    [winsdk_version] : full Windows 10 SDK number (e.g. 10.0.10240.0) or "8.1" to use the Windows 8.1 SDK.   
    [vc_version] : {none} for default VS 2017 VC++ compiler toolset |
                   "14.0" for VC++ 2015 Compiler Toolset |
                   "14.1x" for the latest 14.1x.yyyyy toolset installed (e.g. "14.11") |
                   "14.1x.yyyyy" for a specific full version number (e.g. 14.11.25503)
    [spectre_mode] : {none} for default VS 2017 libraries without spectre mitigations |
                     "spectre" for VS 2017 libraries with spectre mitigations

The store parameter sets environment variables to support Universal Windows Platform application 
development and is an alias for 'uwp'.

For example:
    vcvarsall.bat x86_amd64
    vcvarsall.bat x86_amd64 10.0.10240.0
    vcvarsall.bat x86_arm uwp 10.0.10240.0
    vcvarsall.bat x86_arm onecore 10.0.10240.0 -vcvars_ver=14.0
    vcvarsall.bat x64 8.1
    vcvarsall.bat x64 store 8.1

Please make sure either Visual Studio or C++ Build SKU is installed.
```

It happens everywhere VS2017 is used. It happened here and in the Qt recipe. I've uploaded an older version to conda-forge to workaround this.

**Comment by isuruf** [comment]
Full log at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=73900

**Pull request merged**